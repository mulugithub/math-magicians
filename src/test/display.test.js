import React from 'react';
import { render } from '@testing-library/react';
import DisplayResult from '../components/Display';

describe('Display result component', () => {
  test('render result correctly', () => {
    const { container } = render(<DisplayResult />);
    expect(container).toMatchSnapshot();
  });
});
