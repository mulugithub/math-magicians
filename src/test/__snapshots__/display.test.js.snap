// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display result component render result correctly 1`] = `
<div>
  <div
    class="display"
  >
    <p>
      0
    </p>
  </div>
</div>
`;
